apps/prairielearn/src/question-servers/index.ts

```diff
@@ -5,7 +5,7 @@ import { type QuestionServer } from './types.js';
 export * from './types.js';
 
 type QuestionType = Question['type'];
-type EffectiveQuestionType = 'Calculation' | 'Freeform';
+export type EffectiveQuestionType = 'Calculation' | 'Freeform';
```

Comment:
This won't need to be exported once https://github.com/PrairieLearn/PrairieLearn/pull/12614 lands.